cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -368,6 +368,107 @@ objects:
         }
       }
 
+# TODO: parameterize data.platform-identities.yaml after templatize refactor and helm chart conversion
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: azure-operators-managed-identities-config
+    namespace: ${NAMESPACE}
+  data:
+    azure-operators-managed-identities-config.yaml: |
+      controlPlaneOperatorsIdentities:
+
+        cloud-controller-manager:
+          minOpenShiftVersion: 4.17
+          azureRoleDefinitionResourceId: '/subscriptions/1d3378d3-5a3f-4712-85a1-2485495dfc4b/providers/Microsoft.Authorization/roleDefinitions/ebe170ec-1247-536a-86d9-74c829dd9844'
+          azureRoleDefinitionName: 'Azure Red Hat OpenShift Cloud Controller Manager - Dev'
+          optional: false
+
+        ingress:
+          minOpenShiftVersion: 4.17
+          azureRoleDefinitionResourceId: '/subscriptions/1d3378d3-5a3f-4712-85a1-2485495dfc4b/providers/Microsoft.Authorization/roleDefinitions/589ca160-4fac-501e-ad6c-006a19583727'
```

Comment:
We shouldn't include the `/subscription` information in here in my opinion, CS should append the information here because it's always relevant to the "remote" subscription.  

@machi1990 -> can we capture this work somewhere?  Whenever we create a resource group client we'll need to append the `/subscriptions/<customer-sub-id>` to the front of the roleDefinition.

Reply:
This is a good point you are bringing because i was wondering what is this azure role definition id.
- is it a built-in role definition (akin to "Contributor" or "Owner" ) or a custom one?
- Who creates them? 
- Where are they created? On the customer's account

Is it that the customer will create them on the customer's subscription during cluster creation or these are pre-created roles definition with a bigger scope that are going to be created by us during rollout in a region?

Reply:
The roles are custom roles deployed to the dev sub.

Reply:
Update: when creating role assignments using the azure-sdk-for-go, you don't need to specify `/subscriptions/<sub-id>` as part of the role assignment resourceId.  The Authorization RP will infer it based on the scope you attempt to create it.  

This works for both custom roles and built-in roles.  We can remove `/subscriptions/<sub-id` from all of these.